ooniprobe/Model/JsonResult/TestKeys.h

```diff
@@ -2,6 +2,7 @@
 #import "Tampering.h"
 #import "Simple.h"
 #import "Advanced.h"
+#define BLOCKED @"blocked"
```

Comment:
It's great that writing tests has lead you to do this bit of refactoring. Now that this magic string is a defined constant you are less likely to make typos and maybe miss important bugs.

Reply:
:)